content/en/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -23,7 +23,11 @@ You need to have a Kubernetes cluster and the kubectl command-line tool must be
 
 ### Enable the WindowsRunAsUserName feature gate
```

Comment:
Does this heading still make sense?

Reply:
I guess it's still there if you switch the view to 1.16, so no. Removing